recipes-rubygems/rubygems-activesupport_6.1.1.bb

```diff
@@ -6,16 +6,21 @@ HOMEPAGE = "https://rubyonrails.org"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://MIT-LICENSE;md5=a2e0e281b9b6c47f722da2255eca5d68"
 
-DEPENDS_class-native += "rubygems-concurrent-ruby-native \
-                         rubygems-i18n-native \
-                         rubygems-minitest-native \
-                         rubygems-tzinfo-native \
-                         rubygems-zeitwerk-native"
-RDEPENDS_${PN}_class-target += "rubygems-concurrent-ruby \
-                                rubygems-i18n \
-                                rubygems-minitest \
-                                rubygems-tzinfo \
-                                rubygems-zeitwerk"
+DEPENDS_class-native += " \
```

Comment:
And to be extra picky :-) ` += " ` should be replaced by ` += "` (no leading blank needed)